agents/official/beeai-supervisor/src/supervisor/platform-sdk.ts

```diff
@@ -0,0 +1,115 @@
+import { Client as MCPClient } from "@i-am-bee/acp-sdk/client/index.js";
+import { SSEClientTransport } from "@i-am-bee/acp-sdk/client/sse.js";
+import { Logger } from "beeai-framework";
+
+export class PlatformSdk {
+  private static instance: PlatformSdk;
+
+  protected readonly logger: Logger;
+  private _initialized = false;
+  private _connected = false;
+  private transport: SSEClientTransport;
+  private client: MCPClient;
+  private availableAgents?: string[];
+  private sdkUrl = new URL("http://127.0.0.1:8333/mcp/sse");
```

Comment:
@jezekra1 we would need to provide the URL of the platform through configuration here, can you help out?